ballerina/modules/mock/server.bal

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/http;
+
```

Comment:
Are we planning to use the live server to test this connector? Is this a temporary test suite until full mock services are generated?

Reply:
Yes, this API contains 200 odd endpoints, I am looking for an API with less number of resources and will revamp the test cases to run on both live and mock environments

Reply:
Ohhh..sure